idd/Energy+.idd.in

```diff
@@ -49467,13 +49467,106 @@ Coil:Cooling:Water:DetailedGeometry,
   A7 , \field Condensate Collection Water Storage Tank Name
        \type object-list
        \object-list WaterStorageTankNames
-  N17; \field Design Water Temperature Difference
+  N17, \field Design Water Temperature Difference
        \type real
        \units deltaC
        \minimum> 0.0
        \note This input field is optional. If specified, it is used for sizing the Design Water Flow Rate.
        \note If blank or omitted, the Loop Design Temperature Difference value specified in Sizing:Plant
        \note object is used for sizing the Design Water Flow Rate.
+  N18; \field Design Water Inlet Temperature
+       \type real
+       \units C
+       \minimum> 0.0
+       \note This input field is optional. If specified, it is used for sizing the coil Design Geomtery
+       \note Parameters. If blank or omitted, the Design Loop Exit Temperature value specified in 
+       \note Sizing:Plant object is used for sizing the coil Design Geomtery Parameters.
+
+CoilSystem:Cooling:Water,
```

Comment:
OK. Moved the *CoilSystem:Cooling:Water* documentation before the *CoilSystem:Cooling:DX*.

Reply:
Currently the order will be as follows:

```
\input{src/overview/group-heating-and-cooling-coils}
\input{src/overview/group-coil-system-cooling-water}
\input{src/overview/group-coil-cooling-dx}
```

May be it should be arranged as follows since the the *group-heating-and-cooling-coils* contains the *CoilSystem:Heating:DX*
```
\input{src/overview/group-coil-system-cooling-water}
\input{src/overview/group-coil-cooling-dx}
\input{src/overview/group-heating-and-cooling-coils}
```

Reply:
Not wanting to make extra work for you, but I think it would be better to paste the contents of `group-coil-system-cooling-water.tex` into the middle of `group-heating-and-cooling-coils`.

Reply:
And then what? Delete group-coil-system-cooling-water.tex. Anything else that needs to be done? I thought these tex files were to be somewhat specific to object type.

Reply:
Well, yes. group-coil-system-cooling-water.tex is a new file here and contains only one component, not a whole group, so it would be deleted.  

But, to your point the whole tex file organization is inconsistent. It's not worth lots of effort here, I just think it make more sense to keep the three coilsystem object together. If you want, you could move the other two into group-coil-system-cooling-water.tex and rename it to group-coil-system, but that seems like more trouble than it's worth. I'm probably the only one who cares what the order of the TOC looks like, but I expect to find similar things near each other.

Reply:
The water cooling coil objects `group-heating-and-cooling-coils` are not placed next to each other. So, I may have to re-arrange them before moving the `group-coil-system-cooling-water.tex'.

Reply:
@Nigusse That's not my intent. Just paste the section for CoilSystem:Cooling:Water right before the section for CoilSystem:Cooling:DX. Nothing more.

Reply:
The following is going to be the order of the water coils in `group-heating-and-cooling-coils`:

```
Coil:Cooling:Water
Coil:Cooling:Water:DetailedGeometry
CoilSystem:Cooling:Water:HeatExchangerAssisted
CoilSystem:Cooling:Water
```